magnus_account_invoice_import/models/account_invoice_import.py

```diff
@@ -81,7 +81,10 @@ def invoice_import_direct(self, invoice_b64, invoice_filename, raise_if_fail=Tru
                 partner = self.env.ref('account_invoice_import_ml.unknown_supplier')
 
             invoice_id = self.env['account.invoice'].message_new(
-                msg_dict, custom_values=dict(partner_id=partner.id),
+                msg_dict, custom_values=dict(
+                    partner_id=partner.id,
+                    type='in_invoice',
```

Comment:
what do you think about

```suggestion
                    type='in_invoice',
                    reference=msg_dict.get('subject'),
```

?